### DISCUSSION TITLE
Self-nomination for Contributor role

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#139] [Python Team]
Hi @paketo-buildpacks/python-maintainers,

I would like to nominate myself as a contributor. I have worked closely with @thitch97 - pairing on the [RFC to support Poetry](https://github.com/paketo-buildpacks/rfcs/pull/132) (both the RFC itself and also the background engineering research to determine what the behavior would look like). I am also currently pairing with @thitch97 on debugging the test failures that occur when [bumping to a new version of pipenv](https://github.com/paketo-buildpacks/pipenv/pull/162).

Working independently, I've opened a [set of issues](https://github.com/paketo-buildpacks/python/issues/399) to enable the Python buildpacks to conform to the [SBOM RFC](https://github.com/paketo-buildpacks/rfcs/blob/932d89a970bea06c002496dbd6c7e3812ccab788/text/0038-cdx-syft-sbom.md) and I've started submitting [PRs](https://github.com/paketo-buildpacks/cpython/pull/289) to make those changes.

I believe that I have demonstrated the commitment and ability to contribute, and I'd like to continue to do so as a contributor.

Thanks!
Rob

**Comment by thitch97** [comment]
I support @robdimsdale's self-nomination! As outlined above, he has made helpful technical contributions to the Python buildpacks and is a regular presence at the Paketo Buildpacks working group meetings.

**Comment by ryanmoran** [comment]
Congrats @robdimsdale!

**Comment by robdimsdale** [comment]
thanks folks! :)
